src/README_OLD.md

```diff
@@ -0,0 +1,203 @@
+
+### Get an instance of the editor on your page
+
+As you will see, there is no editor yet in your application, therefore you'll need to give the SDK the command to load the editor dynamically in your html. The `editorId` you provided or the default one, will be used as a target of where the editor will be injected, so you need to create an (preferably) empty `<div>` with the same id as used for editorId or the default id.
+
+Example in `index.html`:
+
+```html
+{# all of the html above #}
+<h1>My first integration</h1>
+
+<div id="chili-editor-example"></div>
+
+{# all of your custom html #}
+
+<script src="https://path/to/sdk/latest/main.js"></script>
+{# all of the html below #}
+```
+
+The SDK should also be aware that there is now room in the html for injecting the editor, so you should call the `loadEditor` method in your integration script.
+
+Example in `integration.js`:
+
+```javascript
+// at the bottom
+window.SDK.loadEditor();
+```
+
+If you now refresh your page in the browser you will see And like magic an instance of the editor gets loaded on your very own page.
+
+<br/>
+
+### Retrieving data from the editor
+
+For retrieving the data, there are 2 ways. For starters we can retrieve data using global listeners, that are configured on initialisation of the SDK.
+
+Below you can find an example for tools:
+
+```javascript
+const sdk = new EditorSDK({
+    onSelectedToolChanged: (tool) => {
+        // here you can do stuff with the newly selected tool
+    },
+    editorLink: 'https://link/to/the/editor', // Will be provided
+    editorId: 'chili-editor-example', // if not provided, default will be 'chili-editor'
+});
+```
+
+Another way is to retrieve state using a getter. For now only in VariableController there are already some getters available, but in the future we'll provide lots of others on all different aspects.
+That way we have multiple ways of retrieving data from the current state.
+
+```javascript
+// Retrieve a list of all variables
+window.SDK.variable.getVariableList();
+```
+
+To check all ways of accessing data, check the controllers in detail, there you see every method available.
+
+<br/>
+
+### Sending data to the editor
+
+Sending data is fairly easy, you just run a method from a controller, with or without parameters.
+
+Below you find an example on changing the tool:
+
+```javascript
+// Set the Hand tool
+window.SDK.tool.setHandTool();
+```
+
+Or an example with a parameter:
+
+```javascript
+// Add a new layout whom inherits from layout with ID 2
+window.SDK.layout.addLayout(2);
+```
+
+<br/>
+
+### Recap
+
+So to recap, you first need to initialise the SDK and Editor, then add an empty div with the ID that you want to pass through the SDK to inject the Editor in.
+Then you're ready to go, if you initialise some listeners with callbacks or use a getter you can easily retrieve data on change or on request.
+To send data you just call the sdk method, with or without parameters (depends on the method).
```

Comment:
I want to leave this for now, since it can come in handy in further documentation steps

Reply:
Done